apps/taxes/src/pages/api/webhooks/checkout-calculate-taxes.ts

```diff
@@ -0,0 +1,27 @@
+import { UntypedCalculateTaxesDocument } from "../../../../generated/graphql";
+import { saleorApp } from "../../../../saleor-app";
+import { createLogger } from "../../../lib/logger";
+import { withTaxesWebhook } from "../../../lib/saleor/with-taxes-webhook";
+import { ActiveTaxProvider } from "../../../modules/taxes/active-tax-provider";
+import { TaxSaleorSyncWebhook } from "../../../modules/taxes/tax-webhook";
+
+export const checkoutCalculateTaxesSyncWebhook = new TaxSaleorSyncWebhook({
+  name: "CheckoutCalculateTaxes",
+  apl: saleorApp.apl,
+  syncEvent: "CHECKOUT_CALCULATE_TAXES",
+  subscriptionQueryAst: UntypedCalculateTaxesDocument,
+  webhookPath: "/api/webhooks/checkout-calculate-taxes",
+});
+
+const handler = withTaxesWebhook(async (payload, config, res) => {
+  const logger = createLogger({});
+  logger.info("Inside CHECKOUT_CALCULATE_TAXES handler");
```

Comment:
```suggestion
  const logger = createLogger({fn: "Checkout calculate taxes handler"});
  logger.info("Handler called");
```

Additional argument added to logger creation will add context to all messages, fo you'll be able to skip repeating name of the function